packages/gasket-plugin-nextjs/lib/index.d.ts

```diff
@@ -4,7 +4,7 @@ import type { NextServer } from 'next/dist/server/next';
 import type { Application } from 'express';
 import type { Fastify } from 'fastify';
 import type { Gasket } from '@gasket/core';
-import type { CreateContext } from 'create-gasket-app';
+import type { CreateContext, GasketPrompt } from 'create-gasket-app';
```

Comment:
Seeing these type imports, what about `CreatePrompt` as the alias? We will not be extending this with Gasket plugins, so that may make more sense.